tls/s2n_cipher_preferences.c

```diff
@@ -1010,9 +1023,32 @@ int s2n_find_cipher_pref_from_version(const char *version, const struct s2n_ciph
 int s2n_config_set_cipher_preferences(struct s2n_config *config, const char *version)
 {
     GUARD(s2n_find_cipher_pref_from_version(version, &config->cipher_preferences));
+    config->minimum_protocol_version = config->cipher_preferences->minimum_protocol_version;
     return 0;
 }
 
+int s2n_config_set_min_protocol_version(struct s2n_config *config, const char *version)
```

Comment:
Why did you choose to use a string identifier instead of integers? There are already definitions of TLS versions available: https://github.com/awslabs/s2n/blob/master/docs/USAGE-GUIDE.md#preprocessor-macros.

Reply:
Good point! Done